docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    command: >
+      sh -c "
+      python manage.py makemigrations &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000
+      "
+    volumes:
+      - ./:/app
```

Comment:
Using a bind mount for the entire project directory (`./:/app`) can lead to potential issues with file permissions and security. Consider using a more specific volume or bind mount only necessary directories.